conversion/indico_conversion/blueprint.py

```diff
@@ -7,9 +7,12 @@
 
 from indico.core.plugins import IndicoPluginBlueprint
 
-from indico_conversion.conversion import RHConversionCheck, RHConversionFinished
+from indico_conversion.conversion import RHConversionCheck, RHDoconverterFinished, RHCloudConvertFinished
 
 
 blueprint = IndicoPluginBlueprint('conversion', __name__)
-blueprint.add_url_rule('/conversion/finished', 'callback', RHConversionFinished, methods=('POST',))
+blueprint.add_url_rule('/conversion/doconverter/finished', 'doconverter-callback',
```

Comment:
I think we use underscores in all other endpoint names (it's just in URLs where they are ugly)